pgrx/src/bgworkers.rs

```diff
@@ -597,15 +597,20 @@ impl BackgroundWorkerBuilder {
     }
 
     /// Once properly configured, call `load_dynamic()` to get the BackgroundWorker registered and started dynamically.
-    pub fn load_dynamic(self) -> DynamicBackgroundWorker {
+    /// Start up might fail, e.g. if max_worker_processes is exceeded. In that case an Err is returned.
+    pub fn load_dynamic(self: Self) -> Result<DynamicBackgroundWorker, ()> {
         let mut bgw: pg_sys::BackgroundWorker = (&self).into();
         let mut handle: *mut pg_sys::BackgroundWorkerHandle = null_mut();
 
-        unsafe {
-            pg_sys::RegisterDynamicBackgroundWorker(&mut bgw, &mut handle);
+        let success = unsafe {
+            pg_sys::RegisterDynamicBackgroundWorker(&mut bgw, &mut handle)
```

Comment:
Please add a `// SAFETY` comment to this block so that the preconditions and postconditions of using this function are documented someways.